examples/style-swin-gan/styleswin_tt.ipynb

```diff
@@ -391,18 +391,29 @@
    "outputs": [],
    "source": [
     "latent_dim = 512\n",
-    "# n_epochs = 500\n",
+    "\n",
+    "\n",
     "n_epochs = 2\n",
-    "batch_size = 24\n",
+    "# n_epochs = 500\n",
+    "\n",
+    "# Hi Sean\n",
+    "#I tried to decrease batch size to avoid OOM but seems not working!\n",
+    "#Any other suggestions, then maybe gradient accumulation will be useful. Thanks, --Miguel\n",
+    "batch_size = 6\n",
+    "# batch_size = 12 # <OOM\n",
+    "# batch_size = 24 # <OOM\n",
```

Comment:
Hi @seansunn , can you please help with this one?

Reply:
Hi @mxochicale , yes the training takes a lot of memory; I'm working on cut down the memory usage and simplify the net; will update the checkpoint as well.

Reply:
Hi @seansunn many thanks Sean, looking forward for your updates.

Reply:
Hi @mxochicale , I've found out that there was an issue in the previous implementation of the APA in the training loop, and I've corrected the implementation in this update.

Note: I've changed the path for the training dataset and commented the previous path; you may need to change it back to run the notebook.

Loading the model to GPU memory should take about 1.5Gb, and applying gradient accumulation for the training loop reduced memory usage a lot: from **~24Gb** (batch size 32) to **~6Gb** (batch size 4 x 8).